lib/receive.dart

```diff
@@ -90,29 +62,45 @@ class ActiveSession extends FfiActiveSession {
     }
   }
 
+  String id() {
+    return _ffiReceiver.id();
+  }
+
   /// The contents of the `&pj=` query parameter including the base64url-encoded public key receiver subdirectory.
   /// This identifies a session at the payjoin directory server.
-  @override
   Future<Url> pjUrl() async {
-    final res = await super.pjUrl();
+    final res = await _ffiReceiver.pjUrl();
     return Url.fromStr(res.asString());
   }
 
-  @override
   PjUriBuilder pjUriBuilder() {
-    final res = super.pjUriBuilder();
+    final res = _ffiReceiver.pjUriBuilder();
     return PjUriBuilder(internal: res.internal);
   }
+
+  String toJson() {
+    return _ffiReceiver.toJson();
+  }
+
+  static Receiver fromJson(String json) {
+    try {
+      final res = FfiReceiver.fromJson(json: json);
+      return Receiver._(ffiReceiver: res.field0);
```

Comment:
I get an error when attempting to deserialize: `type 'ReceiverImpl' is not a subtype of type 'FfiReceiver'`

I think this should be:

```suggestion
      return Receiver._(ffiReceiver: res);
```